log/log.go

```diff
@@ -0,0 +1,63 @@
+package log
```

Comment:
Yea, I think moving it to runtime is a good idea.  There's just nothing really here to make it it's own package.